src/components/ArtifactInfoMetadata/ArtifactInfoMetada.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './artifactinfometadata.scss'
+
+const ArtifactInfoMetadata = ({ item }) => {
+  const { primaryKey } = item.schema
+  let metadata = item.schema.fields.map(_item => {
+    const { name, type } = _item
+    return {
+      name: name,
+      type: type,
+      primary: primaryKey[0] === name ? 'Yes' : '',
```

Comment:
Should not be just the first item in `primaryKey` but if it is included in it.

```suggestion
      primary: primaryKey.includes(name) ? 'Yes' : '',
```